math/mathcore/inc/Fit/EvaluatePoissonLogL.hxx

```diff
@@ -0,0 +1,425 @@
+// @(#)root/mathcore:$Id$
+// Author: L. Moneta Tue Nov 28 10:52:47 2006
+
+/**********************************************************************
+ *                                                                    *
+ * Copyright (c) 2006  LCG ROOT Math Team, CERN/PH-SFT                *
+ *                                                                    *
+ *                                                                    *
+ **********************************************************************/
+
+#ifndef ROOT_Fit_EvaluatePoissonLogL
+#define ROOT_Fit_EvaluatePoissonLogL
```

Comment:
This header looks extremely similar to `EvaluateLogL.hxx`. Can't we make at least some of the code common to both?

Reply:
Maybe. But the structure of the fitting classes is complicated enough as it is. Maybe moving refactoring into FitUtil...